apps/forklift/lib/forklift/data_writer.ex

```diff
@@ -56,6 +56,18 @@ defmodule Forklift.DataWriter do
     end
   end
 
+  @impl Pipeline.Writer
+  def delete(dataset) do
+    endpoints = Application.get_env(:forklift, :elsa_brokers)
+    topic = "#{Application.get_env(:forklift, :input_topic_prefix)}-#{dataset.id}"
+
+    [endpoints: endpoints, topic: topic]
+    |> @topic_writer.delete()
```

Comment:
Maybe for consistency we should just pass in the dataset?  Seems like it would make sense for `TopicWriter` to be able to figure out the endpoints and topic based on the dataset.